src/Pages/gunwookim/Main/Sections/Left/LeftSection.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import '../Sections.scss';
+
+import Feeds from './Feeds/Feeds';
+
+class LeftSection extends React.Component {
+  state = {
+    content: '',
+    feedList: [],
+  };
+
+  handleInput = e => {
+    const { name, value } = e.target;
+
+    this.setState({
+      [name]: value,
+    });
+
+    if (e.key === 'Enter') this.addFeedList();
+  };
+
+  addFeedList = () => {
+    let { content } = this.state;
+    if (content.trim() === '') return;
+
+    this.setState({});
+  };
+
+  componentDidMount() {
+    fetch('http://localhost:3000/gunwookimData/feedsData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(feedList => {
+        this.setState({
+          feedList,
+        });
+      });
```

Comment:
```suggestion
    fetch('/gunwookimData/feedsData.json')
      .then(res => res.json())
      .then(feedList => this.setState({ feedList }));
```
- public 경로에 있는 데이터를 가져올 때는 url과 포트 번호를 생략해주어도 됩니다
- get method는 디폴트 값이라 생략해도 무방할거 같아요!